Sources/DistributedActors/Pattern/WorkerPool.swift

```diff
@@ -222,7 +222,7 @@ public struct WorkerPoolRef<Message: ActorMessage>: _ReceivesMessages {
         self._ref.tell(.forward(message), file: file, line: line)
     }
 
-    public func ask<Answer: ActorMessage>(
+    func ask<Answer: ActorMessage>(
```

Comment:
👍 

And a ticket about making workerpool for DA and dropping the ref based one: https://github.com/apple/swift-distributed-actors/issues/891